exercises/practice/sublist/.meta/Sublist.example.ps1

```diff
@@ -0,0 +1,59 @@
+Enum Sublist {
+    EQUAL
+    UNEQUAL
+    SUBLIST
+    SUPERLIST
+}
+Function Invoke-Sublist() {
+    <#
+    .SYNOPSIS
+    Determine the relationship of two arrays.
+
+    .DESCRIPTION
+    Given two arrays, determine the relationship of the first array relating to the second array.
+    There are four possible categories: EQUAL, UNEQUAL, SUBLIST and SUPERLIST.
+    Note: This exercise use Enum values for return.
+    
+    .PARAMETER Arr1
```

Comment:
I think using something like `Data1` and `Data2` could be a bit more descriptive.